go.mod

```diff
@@ -7,9 +7,13 @@ require (
 	github.com/aws/aws-sdk-go v1.31.2
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/phayes/freeport v0.0.0-20180830031419-95f893ade6f2
+	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.2.0
 	github.com/smartystreets/goconvey v1.6.4
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/viper v1.6.3
-	gopkg.in/yaml.v2 v2.2.4
+	gopkg.in/yaml.v2 v2.2.8
+	k8s.io/api v0.15.12
```

Comment:
Because these k8s package versions can be a headache, I've seen packages put explicit `replace` statements to prevent conflicts. But maybe k8s finally adopted modules properly and this is no longer necessary?

Reply:
Ah, interesting... do you mean something like this? https://github.com/kubernetes/client-go/issues/551#issuecomment-475465546

I'll do some more reserach to see if it seems like that's necessary.